e2e/test/scenarios/admin/people/group-managers.cy.spec.js

```diff
@@ -37,13 +37,21 @@ describeEE("scenarios > admin > people", () => {
       });
 
       // Edit group name
-      cy.icon("ellipsis").eq(0).click();
-      cy.findByText("Edit Name").click();
-      cy.get("input").type(" updated");
-      cy.button("Done").click();
+      cy.findByTestId("admin-content-table").within(() => {
+        cy.icon("ellipsis").eq(0).click();
+      });
+      popover().within(() => {
+        cy.findByText("Edit Name").click();
+      });
```

Comment:
When you only need to do something with just one element, there's no need for a `within`. You can just do:

```js
popover().findByText("Edit Name").click();
```